src/components/TextbitRoot/TextbitContext.tsx

```diff
@@ -81,9 +88,10 @@ const reducer = (state: TextbitProviderState, action: Partial<TextbitProviderSta
 
 
 // Create the context provider component
-export const TextbitContextProvider = ({ children, verbose, autoFocus, debounce, placeholder, placeholders }: PropsWithChildren & {
+export const TextbitContextProvider = ({ children, verbose, autoFocus, onBlur = undefined, debounce, placeholder, placeholders }: PropsWithChildren & {
```

Comment:
Do we need to set undefined here? Other than to really make it clear? Or do we need to differentiate from other falsy values? Just curious really :slightly_smiling_face:

Reply:
ha ha, no. I'll clean it up.